go/parquet/pqarrow/file_writer.go

```diff
@@ -134,13 +134,19 @@ func (fw *FileWriter) RowGroupTotalBytesWritten() int64 {
 	return 0
 }
 
-// WriteBuffered allows to write records and decide where to break your row group
-// based on the TotalBytesWritten rather than on the max row group len.
-// If using Records, this should be paired with NewBufferedRowGroup,
-// while Write will always write a new record as a row group in and of itself.
+// WriteBuffered will either append to an existing row group or create a new one
+// based on the record length and RowGroupTotalBytesWritten.
```

Comment:
`WriteBuffered` itself never checks `RowGroupTotalBytesWritten`, unless i'm missing something.

Reply:
fixed a typo in 25d51d5a58e86ca527d31547ec21e6878e7a852f